src/class-ads.php

```diff
@@ -226,9 +260,9 @@ public function reinsert_variable( $content ) {
 	 * @param string $ad_tag The ad tag html code.
 	 * @param string $key_value_string The key value string.
 	 * @param string $list_ids The list ids.
-	 * @return string
+	 * @return mixed
```

Comment:
```suggestion
	 * @return string|array|string[]|null The modified ad tag or null in case of failure.
```

Reply:
I'm not sure why I had that as `mixed` – thank you!